src/components/MemberListSummary/MemberListSummary.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import { text } from '@storybook/addon-knobs';
+import React from 'react';
+import {
+  MAX_USER_TO_DISPLAY,
+  MemberListSummary as MemberListSummaryComponent,
+  MemberSummaryItem,
+} from './MemberListSummary';
+
+export default { title: 'Components/MemberSummaryItem' };
+
+const memberSummaryItems: MemberSummaryItem[] = [
+  {
+    picture: 'images/favicon-32x32.png',
+    avatarLetters: 'Jim',
+    label: 'Jim',
+  },
+  {
+    picture: 'images/favicon-32x32.png',
+    avatarLetters: 'Sarah',
+    label: 'Sarah',
+  },
+  {
+    picture: 'images/favicon-32x32.png',
+    avatarLetters: 'Julius',
+    label: 'Julius',
+  },
+  {
+    picture: 'images/favicon-32x32.png',
+    avatarLetters: 'Kevin',
+    label: 'Kevin',
+  },
+  {
+    picture: 'images/favicon-32x32.png',
+    avatarLetters: 'Rob',
+    label: 'Rob',
+  },
+  {
+    picture: 'images/favicon-32x32.png',
+    avatarLetters: 'Katie',
+    label: 'Katie',
+  },
+];
+
+export const MemberListSummaryLessThanMaxItems = () => (
```

Comment:
Since we already gave the component name in the title we don't need to duplicate it here.
```suggestion
export const LessThanMaxItems = () => (
```

Reply:
Changed for all examples